find_new/src/find_new.py

```diff
@@ -82,10 +110,10 @@ def submit_jobs_for_granule(hyp3, event_id, granule):
     granule_lists.append([granule])
 
     try:
-        neighbors = asf_search.get_nearest_neighbors(granule['granuleName'])
-    except ASFSearchError:
+        neighbors = get_neighbors(granule['granuleName'])
+    except asf_search.ASFSearch4xxError:
         raise GranuleError()
-    except ServerError as e:
+    except asf_search.ASFSearchError as e:
```

Comment:
I think we wanted to catch `asf_search.CMRError` here as well

Reply:
Likely -- looks like it's its own class of exception:
https://github.com/asfadmin/Discovery-asf_search/blob/master/asf_search/exceptions.py#L25-L26

Reply:
Ah, looks like my comment above got buried: https://github.com/ASFHyP3/hyp3-event-monitoring/pull/59#issuecomment-1068363655

Reply:
Oh yep, you're right, it hasn't been released yet: https://github.com/asfadmin/Discovery-asf_search/blob/v3.0.6/asf_search/exceptions.py